tests/build_/test_plugins_configuration.py

```diff
@@ -294,9 +294,12 @@ def test_render_flatpak(self, compose_ids, build_type):
             assert get_plugin(plugins, "postbuild_plugins", "import_image")
 
     @pytest.mark.parametrize('build_type', (BUILD_TYPE_ORCHESTRATOR, BUILD_TYPE_WORKER))
-    def test_render_prod_not_flatpak(self, build_type):
+    @pytest.mark.parametrize('flatpak', (False, True, False))
```

Comment:
It's unnecessary to list `False` twice. This will cause identical tests to run twice.
Same for `flatpak_base_image` parameter below.

Reply:
oh yes good catch :+1: